### PR TITLE
Add more tests for users, resources and study groups

### PR DESCRIPTION
**Pull request by alicoding**
cloes https://github.com/mozilla/science.mozilla.org/issues/351

### CODE REVIEW

**Comment by alicoding** [comment]
@gideonthomas R?

File: app/scienceapi/resources/tests/test_models.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+from faker import Factory as FakerFactory
+
+from scienceapi.resources.models import Resource
+
+faker = FakerFactory.create()
+
+
+class ResourceFactory(factory.DjangoModelFactory):
+    name = factory.LazyAttribute(lambda o: ' '.join(faker.words(nb=4)))
+    link = factory.LazyAttribute(lambda o: '{a}.com'.format(
```
Comment (gideonthomas) [inline app/scienceapi/resources/tests/test_models.py]: '{0}.com'.format(o.name.lower())

File: app/scienceapi/resources/tests/test_views.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            resource.save()
+
+    def test_list_resources_returns_resource_data(self):
+        """
+        Check if we can get a list of resources
+        """
+        response = self.client.get(reverse('resource-list'))
+
+        self.assertEqual(response.status_code, 200)
+        resource_data = json.loads(response.content.decode('utf-8'))
```
Comment (gideonthomas) [inline app/scienceapi/resources/tests/test_views.py]: let's put all the asserts together

File: app/scienceapi/study_groups/tests/test_views.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        for study_group in self.study_groups:
+            study_group.save()
+
+    def test_list_study_groups_returns_study_group_data(self):
+        """
+        Check if we can get a list of study groups
+        """
+        response = self.client.get(reverse('studygroup-list'))
+
+        self.assertEqual(response.status_code, 200)
```
Comment (gideonthomas) [inline app/scienceapi/study_groups/tests/test_views.py]: same

File: app/scienceapi/users/tests/test_views.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        for user in self.users:
+            user.save()
+
+    def test_list_users_returns_user_data(self):
+        """
+        Check if we can get a list of users
+        """
+        response = self.client.get(reverse('user-list'))
+
+        self.assertEqual(response.status_code, 200)
```
Comment (gideonthomas) [inline app/scienceapi/users/tests/test_views.py]: same

**Comment by gideonthomas** [comment]
r+ with nits

**Pull request merged**